lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -735,6 +735,23 @@ OpFoldResult AtenDimOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// AtenIsFloatingPointOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenIsFloatingPointOp::fold(ArrayRef<Attribute> operands) {
+  if (auto tensorType = getOperand().getType().dyn_cast<BaseTensorType>()) {
+    if (tensorType.hasDtype()) {
+      if(tensorType.getDtype().isa<Float64Type>() || 
+         tensorType.getDtype().isa<Float32Type>() ||
+	 tensorType.getDtype().isa<Float16Type>()) {
```

Comment:
Nice! A cleaner (and complete) way is `..Dtype.isa<Float16Type, BFloat16Type, Float32Type, Float64Type>()) `